app/actions/local/channel.ts

```diff
@@ -284,11 +284,10 @@ export const updateLastPostAt = async (serverUrl: string, channelId: string, las
         return {error: 'not a member'};
     }
 
-    if (lastPostAt > member.lastPostAt) {
-        member.prepareUpdate((m) => {
-            m.lastPostAt = lastPostAt;
-        });
-    }
+    const timestamp = lastPostAt > member.lastPostAt ? lastPostAt : member.lastPostAt;
+    member.prepareUpdate((m) => {
```

Comment:
it is not. We could change this so if they are not different the function returns undefined instead of the model. 0/5 on leaving it like this or making the change

Reply:
changed to only update if needed